.github/workflows/buildsCI.yaml

```diff
@@ -381,18 +381,21 @@ jobs:
         mkdir build
 
     - name: Build
+      # Note: we force RSUSB because, on Linux, the context creation will fail on GHA:
```

Comment:
I vote for adding a dedicated job for DDS and keep this unchanged.
With the tools and V4l.

We can have 20 parallels builds on GHA

Reply:
We do have dedicated DDS jobs...

Reply:
We can make any changes you want, but not as part of this PR, I think

Reply:
But this PR is changing the builds it turn on the RS - USB and turn off the tools
Instead of adding a build with the required flags

Reply:
I don't understand. 
BUILD_TOOLS is on for U20 in U20_ST_Py_EX_CfU_LiveTest. It's not needed for the DDS build.
Same for RSUSB - for DDS, it's not needed. Non-RSUSB is covered by other tests (as well as LibCI).

Reply:
OK